Upgrade to Alpha 45

Here is the changelog:
https://github.com/angular/angular/blob/master/CHANGELOG.md

I thought it would be an easy upgrade, but the tests are now failing

```
Jakes-MacBook-Pro:angular2-seed jakepartusch$ npm test

> angular2-seed@0.0.0 test /Users/jakepartusch/angular2-testing-workspace/angular2-seed
> gulp test

[22:06:38] Requiring external module ts-node/register
[22:06:40] Using gulpfile ~/angular2-testing-workspace/angular2-seed/gulpfile.ts
[22:06:40] Starting 'test'...
[22:06:40] Starting 'clean.test'...
[22:06:40] Finished 'clean.test' after 4.34 ms
[22:06:40] Starting 'tslint'...
[22:06:41] Finished 'tslint' after 861 ms
[22:06:41] Starting 'build.test'...
[22:06:43] Finished 'build.test' after 1.56 s
[22:06:43] Starting 'karma.start'...
INFO [karma]: Karma v0.12.37 server started at http://localhost:9876/
INFO [launcher]: Starting browser PhantomJS2
INFO [launcher]: Starting browser Chrome
INFO [PhantomJS 2.0.0 (Mac OS X 0.0.0)]: Connected on socket Fu4_4o3GcyS0ndPqtipM with id 67905766
WARN [web-server]: 404: /base/traceur.js
WARN [web-server]: 404: /base/selenium-webdriver.js
PhantomJS 2.0.0 (Mac OS X 0.0.0) ERROR: Error{stack: undefined, line: 740, sourceURL: 'http://localhost:9876/base/node_modules/systemjs/dist/system.src.js?8a7dc2b121cf30cbcb09362dbeab69432cb0db29'}
INFO [Chrome 46.0.2490 (Mac OS X 10.11.0)]: Connected on socket Ng8FHyC4OpQO2-emtipN with id 38729127
WARN [web-server]: 404: /base/traceur.js
ERROR: 'Error: XHR error (404 Not Found) loading http://localhost:9876/base/traceur.js
    at error (http://localhost:9876/base/node_modules/systemjs/dist/system.src.js?8a7dc2b121cf30cbcb09362dbeab69432cb0db29:1020:16)
    at XMLHttpRequest.xhr.onreadystatechange [as _onreadystatechange] (http://localhost:9876/base/node_modules/systemjs/dist/system.src.js?8a7dc2b121cf30cbcb09362dbeab69432cb0db29:1028:13)
    at Zone.run (http://localhost:9876/base/node_modules/zone.js/dist/zone-microtask.js?e1fafbeeb394f8255f39a99f6ddff61c589f16a4:120:17)
    at XMLHttpRequest.zoneBoundFn (http://localhost:9876/base/node_modules/zone.js/dist/zone-microtask.js?e1fafbeeb394f8255f39a99f6ddff61c589f16a4:93:19)'

Finished in 0.002 secs / 0 secs

SUMMARY:
✔ 0 tests completed
WARN [web-server]: 404: /base/selenium-webdriver.js
[22:06:45] 'karma.start' errored after 2.35 s
[22:06:45] Error: 1
    at formatError (/Users/jakepartusch/angular2-testing-workspace/angular2-seed/node_modules/gulp/bin/gulp.js:169:10)
    at Gulp.<anonymous> (/Users/jakepartusch/angular2-testing-workspace/angular2-seed/node_modules/gulp/bin/gulp.js:195:15)
    at emitOne (events.js:82:20)
    at Gulp.emit (events.js:169:7)
    at Gulp.Orchestrator._emitTaskDone (/Users/jakepartusch/angular2-testing-workspace/angular2-seed/node_modules/orchestrator/index.js:264:8)
    at /Users/jakepartusch/angular2-testing-workspace/angular2-seed/node_modules/orchestrator/index.js:275:23
    at finish (/Users/jakepartusch/angular2-testing-workspace/angular2-seed/node_modules/orchestrator/lib/runTask.js:21:8)
    at cb (/Users/jakepartusch/angular2-testing-workspace/angular2-seed/node_modules/orchestrator/lib/runTask.js:29:3)
    at removeAllListeners (/Users/jakepartusch/angular2-testing-workspace/angular2-seed/node_modules/karma/lib/server.js:215:7)
    at Server.<anonymous> (/Users/jakepartusch/angular2-testing-workspace/angular2-seed/node_modules/karma/lib/server.js:226:9)
[22:06:45] 'test' errored after 4.78 s
[22:06:45] Error in plugin 'run-sequence'
Message:
    An error occured in task 'karma.start'.
npm ERR! Test failed.  See above for more details.
```

--- a/karma.conf.js
+++ b/karma.conf.js
@@ -20,6 +20,7 @@ module.exports = function(config) {
       'node_modules/zone.js/dist/jasmine-patch.js',
       'node_modules/es6-module-loader/dist/es6-module-loader.js',
       'node_modules/traceur/bin/traceur-runtime.js', // Required by PhantomJS2, otherwise it shouts ReferenceError: Can't find variable: require
+      'node_modules/traceur/bin/traceur.js',
       'node_modules/systemjs/dist/system.src.js',
       'node_modules/reflect-metadata/Reflect.js',
 
@@ -34,6 +35,7 @@ module.exports = function(config) {
 
     // list of files to exclude
     exclude: [
+      'node_modules/angular2/**/*_spec.js'
     ],
 
 

--- a/package.json
+++ b/package.json
@@ -74,7 +74,7 @@
     "zone.js": "^0.5.8"
   },
   "dependencies": {
-    "angular2": "2.0.0-alpha.44",
+    "angular2": "2.0.0-alpha.45",
     "bootstrap": "^3.3.5",
     "es6-module-loader": "^0.17.8",
     "es6-shim": "^0.33.6",

